src/test/java/com/softserve/academy/spaced/repetition/controller/CardControllerTest.java

```diff
@@ -47,9 +48,71 @@ public void setUp() {
                 .build();
     }
 
+    public static Card createCard(long rating, long deckId) {
```

Comment:
If you can, will be better to format this file to previous format.
As a result, git will not show, that you changed this file)